tests/dopamine/tests/threading_utils_integration_test.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright 2018 The Dopamine Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for `threading_utils` integration with `DQNAgent`."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tempfile
```

Comment:
nit: Best practices is to use a fake instead of the real filesystem: http://go/python/testing/filesystem#use-fake-fileystem-to-replace-one-or-more-modules-that-access-the-filesystem

`pyfakefs` is the open-sourced version of the library the go link refers to.

Reply:
This isn't google3 -- I think it's fine to leave this unchanged pending psc@'s approval.

Reply:
Thanks for sharing - I didn't not about this library, it seems very convenient.

For now I removed the tempfile dependency (which I realized dopamine is not using). I replaced with the default `get_temp_dir` method that I just realized `tf.test.TestCase` has.

I'm not sure how the integration with google3 would work so leaving it open for now.